charts/datadog/templates/_helpers.tpl

```diff
@@ -280,11 +280,46 @@ false
 {{- end -}}
 {{- end -}}
 
+{{/*
+Return true if .Values.existingClusterAgent is fully configured
+*/}}
+{{- define "existingClusterAgent-configured" -}}
+{{- if and .Values.existingClusterAgent.join .Values.existingClusterAgent.serviceName .Values.existingClusterAgent.tokenSecretName -}}
+true
+{{- else -}}
+false
+{{- end -}}
+{{- end -}}
+
+{{/*
+Return true if the ClusterAgent needs to be deployed
```

Comment:
if `existingClusterAgent-configured` the cluster agent doesn't need to be deployed. This flag is more about configuring connection to the cluster agent no?

Reply:
cc @clamoriniere

Reply:
it is to know if the agent needs to contact the cluster-agent

Reply:
cc: @hkaj done